tools/cli/src/main/java/org/apache/carbondata/tool/DataSummary.java

```diff
@@ -226,25 +282,46 @@ private int getColumnIndex(String columnName) {
   private boolean collected = false;
 
   private void printColumnStats(String columnName) throws IOException, MemoryException {
-    out.println();
-    out.println("## Column Statistics for '" + columnName + "'");
+    outPuts.add("");
+    outPuts.add("## Column Statistics for '" + columnName + "'");
     collectStats(columnName);
 
     int columnIndex = getColumnIndex(columnName);
     String[] header = new String[]{"BLK", "BLKLT", "Meta Size", "Data Size",
-        "LocalDict", "DictEntries", "DictSize", "AvgPageSize", "Min%", "Max%"};
+        "LocalDict", "DictEntries", "DictSize", "AvgPageSize", "Min%", "Max%", "Min", "Max"};
 
-    ShardPrinter printer = new ShardPrinter(header);
+    ShardPrinter printer = new ShardPrinter(header, outPuts);
     for (Map.Entry<String, DataFile> entry : dataFiles.entrySet()) {
       DataFile file = entry.getValue();
       for (DataFile.Blocklet blocklet : file.getAllBlocklets()) {
-        String min, max;
+        String min, max, minPercent, maxPercent;
```

Comment:
Now min max will not be present if size is more for example varchar string this case is not handled please check

Reply:
handled